pages/index.js

```diff
@@ -1,18 +1,106 @@
 import React from 'react';
-
-import ProfileCard from '../components/profile-card';
+import ProfileCard from '../src/components/profile-card';
 import styled from 'styled-components';
+import { render } from 'react-dom';
 
 export default function Home() {
   return (
-    <Wrapper>
-      나는 홈이야
-      <ProfileCard />
-    </Wrapper>
+    
+      <Wrapper>
+        <Header>
+          CEOS 프론트엔드 운영진
+        </Header>
+        <Main>
+         {Profile(profiles)}
+        </Main>
+      </Wrapper>
   );
 }
 
+  const profiles = [
+     {
+       id: 1,
+       name: "박효진",
+       age: 23,
+       role: "개발 파트장",
+       univ: "이화여자",
+       major: "컴퓨터공학",
+       phoneNum: "010-5106-6785",
+       email: "gywls517@gmail.com",
+       githubLink: "https://github.com/gywlsp",
+       imageUrl:
+         "https://static.wixstatic.com/media/69e8c2_982cbc3d638e49889d8ba64b3c70a6e3~mv2.jpeg/v1/crop/x_199,y_0,w_543,h_560/fill/w_350,h_347,al_c,q_80,usm_0.66_1.00_0.01/%E1%84%8E%E1%85%AC%E1%84%92%E1%85%AD%E1%84%8C%E1%85%B5%E1%86%AB.webp"
+     },
+     {
+       id: 2,
+       name: "양병훈",
+       age: 25,
+       univ: "서강",
+       role: "운영 팀원",
+       major: "컴퓨터공학",
+       phoneNum: "010-8200-1760",
+       email: "ybh1760@gmail.com",
+       githubLink: "https://github.com/ybh1760",
+       imageUrl:
+         "https://static.wixstatic.com/media/69e8c2_3de533d9d5ee4bbaa5fe0c360e8f6a23~mv2.jpeg/v1/crop/x_0,y_456,w_3024,h_3119/fill/w_351,h_347,al_c,q_80,usm_0.66_1.00_0.01/%E1%84%8B%E1%85%A3%E1%86%BC%E1%84%87%E1%85%A7%E1%86%BC%E1%84%92%E1%85%AE%E1%86%AB.webp"
+     },
+     {
+       id: 3,
+       name: "이수완",
+       age: 22,
+       univ: "이화여자",
+       role: "홍보 파트장",
+       major: "컴퓨터공학",
+       phoneNum: "010-5503-7994",
+       email: "2swan965@gmail.com",
+       githubLink: "https://github.com/lee-soowan65",
+       imageUrl:
+         "https://static.wixstatic.com/media/982853_a188a89325cb4a20b6a487f7e3de5d14~mv2.jpg/v1/crop/x_13,y_1126,w_2998,h_2829/fill/w_351,h_335,al_c,q_80,usm_0.66_1.00_0.01/4A2633DE-3322-4CDC-8F8C-35288D83C43B-500.webp"
+     },
+     {
+       id: 4,
+       name: "최수민",
+       age: 26,
+       univ: "서강",
+       role: "부회장",
+       major: "컴퓨터공학",
+       phoneNum: "010-4413-1261",
+       email: "react@kakao.com",
+       githubLink: "https://github.com/greatSumini",
+       imageUrl:
+        "https://static.wixstatic.com/media/69e8c2_43446e9ac65e4f129083516175259a2b~mv2.jpeg/v1/crop/x_0,y_910,w_3024,h_3122/fill/w_347,h_351,al_c,q_80,usm_0.66_1.00_0.01/%E1%84%8E%E1%85%AC%E1%84%89%E1%85%AE%E1%84%86%E1%85%B5%E1%86%AB.webp"
+     }
+  ] 
+
+  const Profile = () => {
+    
+    profiles.sort((a, b) => a.age - b.age);
+    if(profiles.role != "부회장"){
+    return(   
+        profiles.map((profile,key) => (
+          <ProfileCard info = {profile} key ={profile.id}/>
+        ))
+      );
+    }
```

Comment:
key값 너무 잘 넘겨주셨네요 ㅎㅎ 좋슴다 😄 

1. 체이닝패턴을 사용하면 이렇게 한 객체에 연쇄적으로 메소드를 호출하는 상황에서, 호출을 여러줄에 걸쳐서 나눠쓰지 않아도 돼 코드가 깔끔해져요~ 
- [체이닝 패턴](https://webclub.tistory.com/528)

2. 컴포넌트 렌더링 최적화 관점에서, prop에 객체/배열 리터럴을 전달하는 것은 좋지 않습니다! 비겨 함수를 명시적으로 넘기지 않으면, re-render logic이 Component의 prop들에 대해서 얕은 비교만 수행하기 때문인데요. 참조형식의 특성상 객체 리터럴은 내부에 동일한 값이 저장되어 있더라도 다른 값으로 판단될 위험이 있습니다! 되도록이면 spread operator를 사용해서 변수 하나하나 넘겨주세요 :)
- [배열, 객체 리터럴을 넘기면 안 좋은 이유](https://medium.com/vingle-tech-blog/react-%EB%A0%8C%EB%8D%94%EB%A7%81-%EC%9D%B4%ED%95%B4%ED%95%98%EA%B8%B0-f255d6569849#43bd)
- [javascript의 기본형과 참조형](https://epthffh.tistory.com/entry/Javascript-%ED%95%B5%EC%8B%AC-%EA%B0%9C%EB%85%90-%EC%95%8C%EC%95%84%EB%B3%B4%EA%B8%B0-%E2%80%93-JS-Flow%EA%B8%B0%EB%B3%B8%ED%98%95%EA%B3%BC-%EC%B0%B8%EC%A1%B0%ED%98%95)
- [jsx spread attributes](https://gist.github.com/sebmarkbage/07bbe37bc42b6d4aef81)

```suggestion
    // filter도 추가해보세요 ^.^
    return profiles.sort((a, b) => a.age - b.age).map(profile => (
      <ProfileCard { ...profile } key={ profile.id }/>
    ));
```